docs-2.0/k8s-operator/4.cluster-administration/4.7.security/4.7.1.enable-mtls.md

```diff
@@ -373,19 +382,6 @@ The following provides examples of the configuration file to enable mTLS between
 
 ### Configure `spec.<meta|graph|storage>.config`
 
-To enable mTLS between the client and the Graph service, add the following fields under the `spec.graphd.config` in the cluster configuration file. The paths specified in fields with `*_path` correspond to file paths relative to `/user/local/nebula`. **It's important to avoid using absolute paths to prevent path recognition errors.**
-
-```yaml
-spec:
-  graph:
-    config:
-      ca_client_path: certs/root.crt
-      ca_path: certs/root.crt
-      cert_path: certs/server.crt
-      key_path: certs/server.key
-      enable_graph_ssl: "true"
-```
-
 To enable mTLS between services (Graph, Meta, and Storage), add the following fields under the `spec.metad.config`, `spec.graphd.config`, and `spec.storaged.config` respectively in the cluster configuration file.
```

Comment:
Please state explicitly that the `sslCert` section is also required in this case.

Reply:
done